qiskit/test/mock/backends/tokyo/__init__.py

```diff
@@ -12,5 +12,6 @@
 
 """Mock tokyo backend"""
 
+# from .fake_tokyo import FakeTokyoV2
```

Comment:
We can just delete this since we won't have a fake tokyo v2 in this pr. I f we were to add it we can just do this as part of that

```suggestion
```

Reply:
Thank you. I have deleted that in my latest commit but I forgot to push... Fixed!